lib/dynflow/testing/managed_clock.rb

```diff
@@ -26,6 +26,18 @@ def progress
         end
       end
 
+      def skip_first?(ignored_subject)
```

Comment:
Maybe a name in plural would be better for the argument since we're expecting it to be an array

Reply:
good point! updated